pkg/jsonlog/jsonlogbytes.go

```diff
@@ -5,10 +5,10 @@ import (
 	"unicode/utf8"
 )
 
-// JSONLogBytes is based on JSONLog.
+// JSONLogbytes is based on JSONLog.
 // It allows marshalling JSONLog from Log as []byte
 // and an already marshalled Created timestamp.
-type JSONLogBytes struct {
+type JSONLogbytes struct {
```

Comment:
We usually fix this kind of problem by changing the prefix `JSONLog` to another name, that's the problem for stutters, changing `LogBytes` to `Logbytes` might be just trick to golint, not the right approach I'm afraid.